# HG changeset patch
# User Rik <rik@octave.org>
# Date 1348985422 25200
#      Sat Sep 29 23:10:22 2012 -0700
# Node ID 41467bebc11444f16644d944894a33b48d329108
# Parent  2fa06d83263a087ae4c9ebed66d92249543020d1
Makefile.am: Correct EXTRA_DIST to export bootstrap_gnulib.conf.

* Makefile.am: Correct EXTRA_DIST to export bootstrap_gnulib.conf.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -34,17 +34,17 @@ EXTRA_DIST = \
   COPYING \
   ChangeLog \
   INSTALL \
   INSTALL.OCTAVE \
   NEWS \
   README \
   bootstrap \
   build-aux/bootstrap_gnulib \
-  build-aux/bootstrap.conf \
+  build-aux/bootstrap_gnulib.conf \
   build-aux/mk-opts.pl \
   build-aux/mkinstalldirs \
   build-aux/move-if-change \
   etc/NEWS.1 \
   etc/NEWS.2 \
   etc/NEWS.3 \
   etc/OLD-ChangeLogs/ChangeLog \
   etc/OLD-ChangeLogs/ChangeLog.1 \
